src/components/PasswordInput.vue

```diff
@@ -0,0 +1,102 @@
+<template>
```

Comment:
We have two different aproaches:
- Create a password component. If we do that we'll have to to remove all the unnecessary props such as placeholder and replacing it with a `repeat` prop instead.
- Use only an input component with a type="password" prop as change styles according to it.

I prefer the first option if we have different logic between components and I the second one if we only have to change css